faq.md

```diff
@@ -174,6 +174,31 @@ and are commonly used together in applications.  Storing your data on disk
 using Parquet and reading it into memory in the Arrow format will allow
 you to make the most of your computing hardware.
 
+#### **What is the difference between Apache Arrow and Google Protobuf?**
```

Comment:
I think you should move this to after the discussion of Feather. It interrupts the flow of the discussion from Parquet (a columnar data file format we recommend) to Feather (if there is an Arrow file format, why do you recommend Parquet?), and IIUC "what about protobuf" is more similar of a question to "what about flatbuffers".